src/App.jsx

```diff
@@ -14,33 +15,89 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+const SORT_BY_ALPHA = 'abs';
+const SORT_BY_LENGTH = 'lng';
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+function modify(goods, sortBy, isReversed) {
+  const visibleGoods = [...goods];
 
-      <button type="button" className="button is-warning is-light">
-        Reverse
-      </button>
+  if (sortBy === SORT_BY_ALPHA) {
+    visibleGoods.sort((a, b) => a.localeCompare(b));
+  } else if (sortBy === SORT_BY_LENGTH) {
+    visibleGoods.sort((a, b) => a.length - b.length);
+  }
```

Comment:
Minor, but to optimize the code, you could have added reverse sorting right to the sort criteria


```suggestion
  if (sortBy === SORT_BY_ALPHA) {
    visibleGoods.sort((a, b) => isReversed ? b.localeCompare(a)) : a.localeCompare(b));
  } else if (sortBy === SORT_BY_LENGTH) {
    visibleGoods.sort((a, b) => isReversed ? b.length - a.length : a.length - b.length);
  }
```

And then 
```
if (isReversed && sortBy === null) {
    visibleGoods.reverse();
  }

```